src/content/reference/react/Fragment.md

```diff
@@ -158,9 +158,9 @@ function DateRangePicker({ start, end }) {
 
 ---
 
-### Rendering a list of Fragments {/*rendering-a-list-of-fragments*/}
+### Rendre une liste de Fragments {/*rendering-a-list-of-fragments*/}
 
-Here's a situation where you need to write `Fragment` explicitly instead of using the `<></>` syntax. When you [render multiple elements in a loop](/learn/rendering-lists), you need to assign a `key` to each element. If the elements within the loop are Fragments, you need to use the normal JSX element syntax in order to provide the `key` attribute:
+Voici une situation ou vous avez besoin d'écrire explicitement `Fragment` plutôt que d'utiliser la syntaxe `<></>`. Quand vous [rendez plusieurs éléments dans une boucle](/learn/rendering-lists), vous avez besoin d'assigner une `key` à chaque élément. Si les éléments compris dans une boucle sont des Fragments, vous devez utiliser la syntaxe classique JSX afin de fournir l'attribut `key` :
```

Comment:
```suggestion
Voici un cas de figure où vous devez écrire `Fragment` explicitement plutôt que d'utiliser la syntaxe `<></>`. Quand vous [affichez plusieurs éléments dans une boucle](/learn/rendering-lists), vous avez besoin d'associer une `key` à chaque élément. Si les éléments compris dans la boucle sont des Fragments, vous devez utiliser la syntaxe classique JSX afin de fournir la prop `key` :
```

La VO n'est pas parfaite… Elle utilise "the" au lieu de "some" (le truc reste vrai pour tout fragment dans la liste, même si le reste est constitué d'éléments classique), et elle entretient la confusion (pas mal présente dans les nouvelles docs) entre "prop" et "attribut". Je tends à vouloir préserver le distingo historique qui réserve "attribut" aux éléments natifs et "prop" à ce qui touche à React, **surtout** pour les props techniques que sont `key`, `ref` et `children`.